app/assets/stylesheets/new.scss

```diff
@@ -0,0 +1,28 @@
+#border {
+  border: 2px solid black;
+  margin: auto;
+  padding: 20px 30px;
+}
+
+body {
+  // background-image: url(https://static.pexels.com/photos/6989/city-love-rainbow-nyc.jpg);
+  background-size: cover;
+}
+
+h1 {
+  text-align: center;
+  color: #088da5;
```

Comment:
Color literals like `#088da5` should only be used in variable declarations; they should be referred to via variable everywhere else.